content/en/blog/2019/opentelemetry-governance-committee-explained/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: OpenTelemetry Governance Committee Explained
+linkTitle: Governance Committee Explained
+date: 2019-11-06
+canonical_url: https://medium.com/opentelemetry/opentelemetry-governance-committee-explained-860353baba0
```

Comment:
Is this being shown somewhere? I couldn't spot it right away. It would also be good to make an explicit reference to this. And let @SergeyKanzhelev know that we are copying his blog post :-)

Reply:
thank you for doing it!